xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -1188,10 +1394,23 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <param name="index">The zero-based index of the element to get or set.</param>
+        <summary>Gets or sets the element at the specified index.</summary>
+        <value>The element at the specified index.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This property provides the ability to access a specific element in the collection by using the following syntax: `myCollection[index]`.   
```

Comment:
I guess this remark can stay, it seems to apply both for the parent interface and for this method. What do you think?

Reply:
Yup, I'd vote for this to stay.